config/generate.go

```diff
@@ -56,17 +61,49 @@ func GenerateConfig() error {
 		return err
 	}
 
-	return writeConfig(i)
+	if isMissionControlUIPresent {
```

Comment:
Should we ask admin details first?

Reply:
Don't know. Let's keep it as it is only for now since we are asking project id in the beginning even in the init command.

Reply:
Cool.